server/datastore/mysql/software.go

```diff
@@ -145,7 +145,10 @@ func hostSoftwareInstalledPathsDelta(
 			continue
 		}
 
-		key := fmt.Sprintf("%s%s%s", r.InstalledPath, fleet.SoftwareFieldSeparator, s.ToUniqueStr())
+		key := fmt.Sprintf(
+			"%s%s%s%s%s",
+			r.InstalledPath, fleet.SoftwareFieldSeparator, r.TeamIdentifier, fleet.SoftwareFieldSeparator, s.ToUniqueStr(),
```

Comment:
Is this going to have any negative effects the first time this is deployed, since all signed packages will have different keys than what's currently in the database? Or because we're diffing here we'll get the paths with team ID added and paths without team ID removed when the only paths are the signed ones?

Reply:
No, this will be a-ok because there's no hash stored in the db. This is just a hash used in Go code.